### DISCUSSION TITLE
iOSのSafariで下書き記事をプレビューすると横スクロールが発生することがある

### DISCUSSION THREAD

**Discussion by JunichiIto** [increments/qiita-discussions#943] [要望・アイデア・不具合]
iOSのSafariで下書き記事をプレビューすると横スクロールが発生することがあります。
横スクロールが発生すると記事が読みづらいです。（それにプレビューの意味も薄れてしまう）

https://github.com/user-attachments/assets/03cb4451-75a6-4adc-bb4a-d950e4948cad

もしかすると画像の横幅にあわせて横スクロールが発生しているのかもしれません。

![「Ruby 3 4」を編集 - Qiita](https://github.com/user-attachments/assets/5ab8bd8b-237b-44c5-9f16-4636e24d9db4)

横スクロールが発生しないように（公開後の記事と同じ表示になるように）修正してもらえるとありがたいです。

### 環境
- iPhone 15
- iOS 18.2

**Comment by acronhub** [comment]
@JunichiIto
ご報告ありがとうございます。

社内でも把握している問題点であり、対応を検討しておりますが、優先順位の兼ね合いから着手できておりません🙇🏻‍♂️

具体的な対応時期をお約束することが難しい状況ではありますが、お待ちいただけると幸いです🙏

**Reply by JunichiIto** [reply]
返信ありがとうございます。了解です！

**Reply by makoto-ogata** [reply]
@JunichiIto 
お待たせしました🙇🏻 
モバイルで下書き記事をプレビューすると横スクロールする不具合を修正いたしました。

**Reply by JunichiIto** [reply]
@makoto-ogata ありがとうございます。修正されていることを確認しました。とても見やすくなりました 🙌
